server/src/internalClusterTest/java/org/opensearch/search/nested/SimpleNestedExplainIT.java

```diff
@@ -111,5 +114,7 @@ public void testExplainMultipleDocs() throws Exception {
             .prepareUpdateSettings()
             .setPersistentSettings(Settings.builder().putNull(CLUSTER_CONCURRENT_SEGMENT_SEARCH_SETTING.getKey()).build())
             .get();
+
+        deleteBogusDocs(bogusIds);
```

Comment:
Is this actually needed? It doesn't look like this index is used anywhere else.

Reply:
this is to avoid bogus docs affecting the search for the other tests added in the class as the cluster will be shared.

Reply:
I get that suite scope is the default but there's no other tests in this class and `setupSuiteScopeCluster` is not used to create the indices so it doesn't seem like they're meant to be shared across tests.